eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -197,11 +197,20 @@ abstract class SafeMojo extends AbstractMojo {
     @Parameter(property = "eo.cache")
     protected Path cache = Paths.get(System.getProperty("user.home")).resolve(".eo");
 
+    /**
+     * Used for object versioning implementation.
+     * If set to TRUE, external tojos are used instead of foreign ones and all
+     * inherited Mojos behave a bit differently.
+     * @checkstyle VisibilityModifierCheck (10 lines)
```

Comment:
@maxonfjvipon As I understand, in the future you are going to remove that boolean field, isn't it? If it is so, please, add a puzzle for it.